### PR TITLE
Create CompatHelper.yml

### PR DESCRIPTION
**Pull request by abelsiqueira**

### CODE REVIEW

**Comment by abelsiqueira** [comment]
I've removed FreeBSD < 1.2 because of the seg faults.

File: .cirrus.yml [hunk 1]
```diff
@@ -4,9 +4,6 @@ task:
   name: FreeBSD
   env:
     matrix:
-      - JULIA_VERSION: 1.0
-      - JULIA_VERSION: 1.1
-      - JULIA_VERSION: 1.2
```
Comment (dpo) [inline .cirrus.yml]: Why remove those?

**Comment by dpo** [comment]
https://github.com/JuliaLang/julia/issues/34822

Why don't we allow them to fail instead?

**Comment by abelsiqueira** [comment]
Sure

**Comment by abelsiqueira** [comment]
Now appveyor had a segfault!?

**Comment by dpo** [comment]
@ararslan  Does it look to you as though this is a crash similar to that on FreeBSD I reported yesterday: https://ci.appveyor.com/project/dpo/benchmarkprofiles-jl/builds/30953127/job/yn7qgih01unda797#L121

**Comment by ararslan** [comment]
No, that definitely looks different. That looks like an error in codegen.

**Comment by dpo** [comment]
Hm. First time we see it.

**Comment by ararslan** [comment]
Do you get it consistently if you restart the job? Also it seems to only have happened on 1.1, so perhaps the underlying issue in Julia has been fixed since then.

**Comment by dpo** [comment]
Now it's gone.

**Pull request merged**